config/database.js

```diff
@@ -15,7 +15,6 @@ module.exports = ({ env }) => {
         },
         options: {
           debug: env('debug'),
-          autoMigration: env('DATABASE_AUTOMIGRATION') === 'true',
           pool: {
```

Comment:
@alaa-yahia this was the cause of the problem you had .. I thought that the default for `autoMigration` is false, but it's actually true .. so when you didn't have the env variable, this stopped the creation of the migrations to update the database. This is confusing because the tables still show in Strapi Admin (because the values in the UI come from a different table in the DB for strapi settings and it doesn't matter if the columns and tables are created). Anyhow, I am just got rid of the setting now, because there is no scenario where we'd need to disable autoMigration right now